java-manta-client-unshaded/src/main/java/com/joyent/manta/client/crypto/AesCbcCipherDetails.java

```diff
@@ -121,8 +122,8 @@ static long calculateContentLength(final long plaintextSize, final int blockByte
 
         // CBC requires an IV an extra block for chaining to work
         if (!hasIV && plaintextSize >= blockBytes) {
-            final double blocks = Math.floor(plaintextSize / ((long) blockBytes));
-            calculatedContentLength = (((long)blocks + 1) * blockBytes);
+            final long blocks = Math.floorDiv(plaintextSize, ((long) blockBytes));
```

Comment:
Let's make this a separate PR and commit. This is the type of change that you want it to be highly visible.

Reply:
Noted.